src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -552,6 +552,44 @@ object TezosDatabaseOperations extends LazyLogging {
         Voting.BallotCounts(yaysCount, naysCount, passesCount)
       }
 
+  /** Gets ballot operations for given level */
+  def getBallotOperationsCountForLevel(level: Int)(implicit ec: ExecutionContext): DBIO[Voting.BallotCounts] =
```

Comment:
I fail to see the difference with `getBallotOperationsForLevel` 😅

Reply:
check out the filter part 😉

Reply:
yep, double copied, removed this leftover, good catch!